plerkle/src/account_info_generated.rs

```diff
@@ -0,0 +1,310 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Should autogenerated files be added to version control? I can personally go either way on this. It's way easier to read code if these files are there, but in theory, people operating the repo/running the code will just handle this in the build stage

Reply:
I think it should probably not be in there long term so that our build system will have generated files that match with the version of flatbuffers we believe we are using.

However, at first I wanted to leave the generated files in there because otherwise all users will have to install, build, and setup their path for the flatbuffers compiler, and didn't want to add this minor headache for everyone.

Reply:
Thanks Michael! Yeah should be easier to remove later on than to add